DataFormats/QualityControl/CMakeLists.txt

```diff
@@ -9,30 +9,60 @@
 # granted to it by virtue of its status as an Intergovernmental Organization
 # or submit itself to any jurisdiction.
 
+file(MAKE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/include)
```

Comment:
This is to avoid getting the following error in a fresh build:
```
CMake Error at cmake/O2AddLibrary.cmake:136 (message):
  Trying to append non existing include directory
  /sw/BUILD/e90eeace6d37853812ccc44bea01b8b6c24dc710/O2/DataFormats/QualityControl/include
Call Stack (most recent call first):
  DataFormats/QualityControl/CMakeLists.txt:12 (o2_add_library)
  ```